### PR TITLE
incorporate AC review feedback

### PR DESCRIPTION
**Pull request by igrigorik**
@plehegar @siusin @toddreifsteck @yoavweiss PTAL.

I believe this should capture all of the outstanding feedback and questions from the AC review.

### CODE REVIEW

**Comment by siusin** [comment]
LGTM.

@wseltzer ping? plh is on vacation this week... 

[1] [comments from the AC review](https://www.w3.org/2002/09/wbs/33280/2018-webperf-charter/results).

**Review by yoavweiss — APPROVED** [review]
LGTM

**Comment by wseltzer** [comment]
2 comments: 
1. Should the privacy consideration be stated more strongly?  Instead of "The group reserves the right not to publish a specification if acceptable mitigation strategies cannot be found" say "The group *should not* publish a specification if acceptable mitigation strategies cannot be found." 

2. I don't think the decision policy edit addresses the comment, which asks that Calls for Consensus be called out explicitly. Is that something the group might do, for example by putting "call for consensus" into the subject header when sending minutes of meetings that contain resolutions?

**Comment by igrigorik** [comment]
Thanks for the feedback Wendy.

> Should the privacy consideration be stated more strongly? Instead of "The group reserves the right not to publish a specification if acceptable mitigation strategies cannot be found" say "The group should not publish a specification if acceptable mitigation strategies cannot be found."

Practically speaking, I think they're stating the same thing, but I'm ok with proposed wording. 

> I don't think the decision policy edit addresses the comment, which asks that Calls for Consensus be called out explicitly. Is that something the group might do, for example by putting "call for consensus" into the subject header when sending minutes of meetings that contain resolutions?

The decision here was to tighten our requirements around publishing minutes, and improving our process to get those out in a timely fashion for everyone to review. I'm happy to follow "[dd/mm/yy] meeting minutes / call for consensus" convention for subject lines carrying link to the minutes.

---

@yoavweiss @toddreifsteck any objections?

**Comment by igrigorik** [comment]
Updated to capture the above feedback.

@toddreifsteck @yoavweiss PTAL. If this looks good, I think we're good to merge.

File: index.html [hunk 1]
```diff
@@ -502,7 +502,7 @@ <h2>
...
         </h2>
         <p id="public">
-          Technical discussions for this Working Group are conducted in <a href="https://www.w3.org/2015/Process-20150901/#confidentiality-levels">public</a>. Meeting minutes from teleconference and face-to-face meetings will be archived for public review, and technical discussions and issue tracking will be conducted in a manner that can be both read and written to by the general public. Working Drafts and Editor's Drafts of specifications will be developed on a public repository, and may permit direct public contribution requests.
+          Technical discussions for this Working Group are conducted in <a href="https://www.w3.org/2015/Process-20150901/#confidentiality-levels">public</a>. Meeting minutes from teleconference and face-to-face meetings will be archived for public review (with "[dd/mm/yy] meeting minutes / call for consensus" subject line), and technical discussions and issue tracking will be conducted in a manner that can be both read and written to by the general public. Working Drafts and Editor's Drafts of specifications will be developed on a public repository, and may permit direct public contribution requests.
```
Comment (yoavweiss) [inline index.html]: I think it'd be better if we add the "call for consensus" bit only when publishing minutes where we're actually calling for consensus (e.g. the group decided to adopt a new proposal from incubation).

So maybe something along the lines of "Meeting minutes from teleconference and face-to-face meetings will be archived for public review. When significant decisions were taken during the teleconference, the minutes will be published while including a 'call for consensus' in their subject line."

Comment (igrigorik) [reply index.html]: Good call. Also, realized that I was updating the wrong section. Pushed an update, PTAL.

**Comment by yoavweiss** [comment]
LGTM. Good to merge from my perspective

**Comment by igrigorik** [comment]
Thanks Yoav. 

@toddreifsteck I'll let you do the last pass and merge if this looks reasonable. :)

**Comment by toddreifsteck** [comment]
LGTM. Merging!

**Pull request merged**